packages/@aws-cdk/integ-runner/README.md

```diff
@@ -0,0 +1,276 @@
+  Rerun integration tests if snapshot fails
+- `--clean` (default=`true`)
+  Destroy stacks after deploy (use `--no-clean` for debugging)
+- `--verbose` (default=`false`)
+  verbose logging
+- `--parallel` (default=`false`)
+  Run tests in parallel across default regions
+- `--parallel-regions`
+  List of regions to run tests in. If this is provided then all tests will
+  be run in parallel across these regions
+- `--directory` (default=`test`)
+  Search for integration tests recursively from this starting directory
+- `--force` (default=`false`)
+  Rerun integration test even if the test passes
+
+Example:
+```bash
+integ-runner --update --parallel --parallel-regions us-east-1 --parallel-regions us-east-2 --parallel-regions us-west-2 --directory ./
+```
+
+This will search for integration tests recursively from the current directory and then execute them in parallel across `us-east-1`, `us-east-2`, & `us-west-2`.
+
+### integ.json schema
+
+```ts
+interface CdkCommand {
+  /**
+   * Whether or not to run this command as part of the workflow
+   * This can be used if you only want to test some of the workflow
+   * for example enable `synth` and disable `deploy` & `destroy` in order
+   * to limit the test to synthesis
+   *
+   * @default true
+   */
+  readonly enabled?: boolean;
+
+  /**
+   * Additional arguments to pass to the command
+   * This can be used to test specific CLI functionality
+   *
+   * @default - only default args are used
+   */
+  readonly args?: string[];
+
+  /**
+   * If the runner should expect this command to fail
+   *
+   * @default false
+   */
+  readonly expectError?: boolean;
+
+  /**
+   * This can be used in combination with `expectedError`
+   * to validate that a specific message is returned.
+   *
+   * @default - do not validate message
+   */
+  readonly expectedMessage?: string;
+}
+
+interface CdkCommands {
+  /**
+   * Options to for the cdk deploy command
+   *
+   * @default - default deploy options
+   */
+  readonly deploy?: CdkCommand;
+
+  /**
+   * Options to for the cdk synth command
+   *
+   * @default - default synth options
+   */
+  readonly synth?: CdkCommand;
+
+  /**
+   * Options to for the cdk destroy command
+   *
+   * @default - default destroy options
+   */
+  readonly destroy?: CdkCommand;
+}
+
+interface Hooks {
+  readonly preDeploy?: string[];
+  readonly postDeploy?: string[];
+  readonly preDestroy?: string[];
+  readonly postDestroy?: string[];
+}
+
+interface TestCase {
+  /**
+   * Stacks that should be tested as part of this test case
+   * The stackNames will be passed as args to the cdk commands
+   * so dependent stacks will be automatically deployed unless
+   * `exclusively` is passed
+   */
+  readonly stacks: string[];
+
+  /**
+   * Run update workflow on this test case
+   * This should only be set to false to test scenarios
+   * that are not possible to test as part of the update workflow
+   *
+   * @default true
+   */
+  readonly update?: boolean;
```

Comment:
This `update` property has a different meaning than the `--update` command line arg, right? I wonder if this may cause some confusion.